port: 50000
ip: "192.168.0.50"


d_level0_l0: 0.60
d_level1_l0: 0.30

d_level0_l1: 0.60
d_level1_l1: 0.30

